go.mod

```diff
@@ -50,6 +50,8 @@ require (
 	github.com/go-logr/logr v1.2.3 // indirect
 	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/golang-jwt/jwt/v4 v4.4.2 // indirect
+	github.com/google/subcommands v1.0.1 // indirect
```

Comment:
Can you run `go mod tidy`?  My VS Code was complaining about this when I checked out the PR.